### PR TITLE
To Fix the issue specific to drag selection in non-frozen part of grid if there are multiple grids.

### PR DESCRIPTION
**Pull request by Hunt05**
Issue Description: if we have multiple grids in same page/in the DOM, earlier code looks for  '.grid-canvas-left' width which indeed gets the result of the grid where the event triggered along with the other grids present in the DOM. If the action is done on second grid(which has frozen column) instead of first one, earlier code is getting the width of first grid instead of second one. So made changes to get the width only for the event triggered grid.

### CODE REVIEW

**Comment by Hunt05** [comment]
[IssueExample.zip](https://github.com/6pac/SlickGrid/files/6232338/IssueExample.zip)
PFA for issue example(This .zip contains the HTML file). Try drag and select in second grid. DragSelection is selecting all the columns in non-frozen part of second grid due to columnOffset of first  grid.

**Comment by Hunt05** [comment]
@6pac , @ghiscoding Let me know if any.

File: plugins/slick.cellrangeselector.js [hunk 1]
```diff
@@ -71,13 +71,13 @@
       _isBottomCanvas = _$activeCanvas.hasClass('grid-canvas-bottom');
 
       if (_gridOptions.frozenRow > -1 && _isBottomCanvas) {
-        _rowOffset = (_gridOptions.frozenBottom) ? $('.grid-canvas-bottom').height() : $('.grid-canvas-top').height();
+        _rowOffset = (_gridOptions.frozenBottom) ? $(`.${_grid.getUID()} .grid-canvas-bottom`).height() : $(`.${_grid.getUID()} .grid-canvas-top`).height();
```
Comment (ghiscoding) [inline plugins/slick.cellrangeselector.js:74]: could you please change the syntax to the old ES5 syntax of concatenating text + variable. This new ES6 syntax you have here won't work in legacy browser and we want to keep it as ES5 compatible at the source

**Comment by ghiscoding** [comment]
@Hunt05 
I'm not sure how to test this, am I supposed to something in the UI or in the log on the demo you sent? Can you possible make a print or an animated gif of what that PR fixes? I mean, I'm ok with adding the grid UID to the css classes (with the change I asked on previous line), I just don't understand what this fixes exactly and like I said, I did try your code change

**Review by ghiscoding — CHANGES_REQUESTED** [review]
Please use ES5 syntax to make sure the grid still works in legacy browser

File: plugins/slick.cellrangeselector.js (hunk 1 above)
Comment (Hunt05) [reply plugins/slick.cellrangeselector.js]: @ghiscoding, I have done the changes as per your request.

**Comment by Hunt05** [comment]
@ghiscoding PFA for more details. Can you please let me know when can we expect this fix in future release ?
Print_BeforeChanges
Print_AfterChanges

?

**Pull request merged**